README.md

```diff
@@ -10,9 +10,9 @@
 
 - [azure-active-directory-b2c-spring-boot-starter](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-starter-active-directory-b2c)
 - [azure-active-directory-spring-boot-starter](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-starter-active-directory) 
-- [azure-cosmosdb-spring-boot-starter](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-starter-cosmosdb)
+- [azure-spring-boot-starter-cosmos](https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/spring/azure-spring-boot-starter-cosmos)
```

Comment:
Do we need to use tag instead of main branch?

Reply:
the link in main branch is blank,removed in main branch now

Reply:
Can you merge this PR？@chenrujun